src/reducers/cds-services-reducers.js

```diff
@@ -3,7 +3,7 @@ import * as types from '../actions/action-types';
 
 const initialState = {
   configuredServices: {},
-  defaultUrl: 'https://fhir-org-cds-services.appspot.com/cds-services',
+  defaultUrl: 'http://cds2.interopion.com/cds-services',
```

Comment:
Was this change intentional?

Reply:
nm - I see in your description you say this is a temporary change

Reply:
Yeah, just edited the PR description. We'll change it back when it's done, but this is just to see a card that has a SMART app